uranium/packages/__init__.py

```diff
@@ -55,14 +55,17 @@ def index_urls(self, value):
                  "only lists can be set as a value for indexes")
         self._index_urls = value
 
-    def install(self, name, version=None, develop=False, upgrade=False):
+    def install(self, name, version=None, develop=False, upgrade=False, install_options=[]):
```

Comment:
an empty list is dangerous, in case that value gets modified later. When python evaluates the method signature, it creates a static list, and values that get appended to it will stick. 

I don't see that as a risk in this particular case, but I try to avoid it entirely.

can we use None and populate it with a list instead? that's a standard pattern:

{code}
install_options=None):
install_options = install_options or []
{code}